code/modules/projectiles/guns/projectile/revolver/lemant.dm

```diff
@@ -101,6 +102,35 @@
 	else
 		..()
 
+/obj/item/gun/projectile/revolver/lemant/load_ammo(var/obj/item/A, mob/user)
+	if(istype(A, /obj/item/ammo_magazine))
```

Comment:
This allows reloading form a mag

Reply:
This mean taking out the bullets from the magazine and putting it in the lemant by hand.
Also ammo boxes are a type of magazine.